README.md

```diff
@@ -32,6 +32,12 @@ curl https://ollama.ai/install.sh | sh
 The official [Ollama Docker image `ollama/ollama`](https://hub.docker.com/r/ollama/ollama)
 is available on Docker Hub.
 
+### Autocompletion
+To enable autocompletion generate completion script for your shell (`bash`, `zsh`, `fish`) and `source` it in your shell config (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, `~/.bashrc`, etc.):
+```
+echo "source <(./ollama completion bash)" >> ~/.bashrc # Required bash-completion package installed
```

Comment:
You have instructions for bash, but not zsh and fish.

Reply:
Added an example for fish and bash 👍